evergreen/evergreen.yml

```diff
@@ -60,15 +60,13 @@ functions:
 
           export MONGO_ORCHESTRATION_HOME="$DRIVERS_TOOLS/.evergreen/orchestration"
           export MONGODB_BINARIES="$DRIVERS_TOOLS/mongodb/bin"
-          export CRYPT_SHARED_LIB_PATH="$CRYPT_SHARED_LIB_PATH"
```

Comment:
We do not have the value yet. It will be populated after executing `bootstrap-mongo-orchestration`